openstack/cdn/v1/domains/requests.go

```diff
@@ -11,6 +11,10 @@ type ExtensionOpts struct {
 	EnterpriseProjectId string `q:"enterprise_project_id"`
 }
 
+type PrivateBucketAccess struct {
+	Status bool `json:"status"`
+}
```

Comment:
it's better to define a request struct with suffix **Opts**.
```suggestion
type PrivateBucketAccessOpts struct {
	Status *bool `json:"status,omitempty"`
}
```